### PR TITLE
Use correct punchitem.itemNo arg for PublishPunchItemCreatedIntegrati…

### PR DESCRIPTION
**Pull request by Hatleskog**
…onEvents

### CODE REVIEW

File: src/Equinor.ProCoSys.Completion.Command/PunchItemCommands/DuplicatePunchItem/DuplicatePunchItemCommandHandler.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public async Task<List<GuidAndRowVersion>> Handle(DuplicatePunchItemCommand requ
                 punchCopyProperties.Insert(0, new Property(nameof(PunchItem.ItemNo), punchItem.ItemNo, ValueDisplayType.IntAsText));
 
                 var integrationEvent = await PublishPunchItemCreatedIntegrationEventsAsync(
-                    punchCopy.ItemNo, 
+                    punchItem.ItemNo,
```
Comment (janid1967) [inline src/Equinor.ProCoSys.Completion.Command/PunchItemCommands/DuplicatePunchItem/DuplicatePunchItemCommandHandler.cs:71]: This change can't be correct!
But at line 68 there is a bug: it should be `new Property(nameof(PunchItem.ItemNo), punchCopy.ItemNo, ValueDisplayType.IntAsText`

**Review by janid1967 — CHANGES_REQUESTED** [review]
Se comment

File: src/Equinor.ProCoSys.Completion.Command/PunchItemCommands/DuplicatePunchItem/DuplicatePunchItemCommandHandler.cs (hunk 1 above)
Comment (janid1967) [reply src/Equinor.ProCoSys.Completion.Command/PunchItemCommands/DuplicatePunchItem/DuplicatePunchItemCommandHandler.cs:71]: Ahh ... now I see it. The first argument of PublishPunchItemCreatedIntegrationEventsAsync is sourceItemNo, used in History. But still, the bug at line 68 should be investigated.

How has then bug been reported? Is there an issue? Should we try to correct already made history or is that "impossible"? 

I guess bug on line 68 has made incorrect data in PCS4?

Comment (janid1967) [reply src/Equinor.ProCoSys.Completion.Command/PunchItemCommands/DuplicatePunchItem/DuplicatePunchItemCommandHandler.cs:71]: I investigate now

**Review by DownGoat — APPROVED** [review]

**Pull request merged**